app/views/petitions/check.html.erb

```diff
@@ -1,8 +1,42 @@
 <% content_for(:page_title) { "Create a new e-petition - e-petitions" } %>
-<% content_for :title do %>
-<div class="title_block">
-  <h1>Create a new e-petition</h1>
-</div>
-<% end %>
+<h1 class="page-title">What action would you like the government to take?</h1>
 
-<%= render "pre_creation_search_form" %>
+<form method="get" action="/petitions/check_results">
+  <div class="form-group">
+    <label for="q" class="form-label visuallyhidden">
+      Action
+    </label>
+    <textarea name="q" id="q" value="<%= @petitions.try(:query) %>" class="form-control" rows="3" maxlength="80"></textarea>
+    <span class="character-count">80</span>
+    <details class="form-group-help">
+      <summary><span class="summary">Help writing good actions</span></summary>
+      <div class="panel-indent">
+        <p>What do you want the government to do and why? Write as briefly and clearly as you can.</p>
+        <p>Here are some examples of well-written titles:</p>
+        <ul class="list-bullet">
+          <li>"End non-stun slaughter to promote animal welfare"</li>
+          <li>"Introduce Black History to the primary curriculum"</li>
+          <li>"Don't increase MPs' salaries - they are well-paid already"</li>
+        </ul>
+        <p>You don't need to say "Petition to...", "Call on the government to..." or "We the undersigned...".</p>
+        <p>Here are some examples of badly-written titles:</p>
+        <ul class="list-bullet">
+          <li>"Jack's law"</li>
+          <li>"Russia Problem on US ruling"</li>
+          <li>"Justice for David Thomas"</li>
+        </ul>
+        <p>In each case, it’s not clear what the government is being asked to do or why.</p>
+        <p>Don't write in all caps. It's harder to read.</p>
+        <p>You’re limited to 80 characters here. You can add more in the next step.</p>
+      </div>
+    </details>
```

Comment:
We repeat this large chunk of helpful info in the petition create page, a good candidate for a partial perhaps?